### DISCUSSION TITLE
'Acquisition' not defined when importing the package

### DISCUSSION THREAD

**Discussion by greg-bioinfo** [BodenmillerGroup/steinbock#176] [Q&A] [answered]
hi,
when i'm trying to import the package imc from steinbock.preprocessing i have this error.
Does anyone know why ?
Thanks
![Capture d’écran 2023-03-14 120032](https://user-images.githubusercontent.com/108455082/224981410-c7b5de62-9365-4b83-aa10-eb2a26161714.jpg)

**Comment by Milad4849** [comment]
Hello, 
I cannot reproduce the error on my side. Please make sure you are using the latest version of steinbock. If the problem persist please provide the the output of `conda list` for the conda environment that you use.

**Reply by greg-bioinfo** [reply]
Thank you for the answer.
I have the same issue on google colab.
I use the version 0.16.0
![Capture d’écran 2023-03-14 135840](https://user-images.githubusercontent.com/108455082/225008300-50e322f2-9cf4-45fa-a426-4d46b3e50dbe.jpg)

**Reply by Milad4849** [reply]
I cannot reproduce the error using steinbock 0.16.0. Could you please provide the your versions of the packages that steinbock requires?

**Reply by greg-bioinfo** [reply]
this ?
![image](https://user-images.githubusercontent.com/108455082/225036177-44c5c169-b832-486c-8c46-5e55562ae65e.png)

**Comment by Milad4849** [comment]
Not exactly, could you please make a conda environment and install steinbock in that environment? Then if the issue persists, paste the output of `conda list` here.

**Reply by greg-bioinfo** [reply]
Thanks for your patience.
I created a new environment and still got the same error. Here is the output of conda list.
![Capture d’écran 2023-03-15 102923](https://user-images.githubusercontent.com/108455082/225267777-aebf4390-5c7d-43f2-bed0-2791cb823a5c.jpg)
![Capture d’écran 2023-03-15 103009](https://user-images.githubusercontent.com/108455082/225267810-ca701953-6168-4b0d-93c3-d553b6ef8deb.jpg)

**Comment by Milad4849** [comment, marked as answer]
Thanks for your reply and raising this issue. Currently the documentation regarding the use of the package in conda environments is incomplete. Here are the steps to use steinbock from a conda environment:
- create a new conda environment with python 3.9/3.8 and pip.
```
conda create -n "myenv" python=3.9 pip
```
- Activate the environment and Install the packages in `requirements.txt` using pip. The file is located at the steinbock root directory.
```
conda activate myenv
pip install -r path/to/requirements.txt
```
This should resolve the error you encounter.

**Reply by greg-bioinfo** [reply]
Perfect it works thank you for your availability

**Comment by super-818** [comment]
The file is located at the steinbock root directory didn‘t have requirements.txt
![image](https://github.com/user-attachments/assets/98bef118-a695-49c1-b8bc-48db7a881ca1)
